src/components/common/containers/DropdownButtonContainer.js

```diff
@@ -1,34 +1,65 @@
 import { connect } from 'react-redux';
```

Comment:
Considering that we use `DropdownButton` without this container, maybe we could move it to components/TextEditor/containers. Most of the logic in here is pretty specific to the TextEditor anyway.